Sources/ETCDExample/ETCDExample.swift

```diff
@@ -13,32 +13,57 @@
 //===----------------------------------------------------------------------===//
 import ETCD
 import NIO
+import Dispatch
 
 @main
 struct Example {
     static func main() async throws {
+        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
Instead of creating this we can just use `MultiThreadedEventLoopGroup.singleton`

Reply:
Updated!